src/services/BrowserActionService.js

```diff
@@ -17,10 +18,13 @@ export const showNumberOfCookiesInIcon = async (tab) => {
 		domain: getHostname(tab.url),
 		storeId: tab.cookieStoreId
 	});
-	browser.browserAction.setBadgeText({
-		text: cookies.length.toString(),
-		tabId: tab.id
-	});
+
+	if (cookies.length !== 0) {
+		browser.browserAction.setBadgeText({
+			text: cookies.length.toString(),
+			tabId: tab.id
+		});
+	}
```

Comment:
I think you probably want an `else` to hide it otherwise, else it'll just stick with whatever the last number was, no?

Reply:
@seansfkelley Thanks for your review.

AFAIK, the badge was hidden even when the `else` statement was missing. The newest commit addresses your review comment.